### DISCUSSION TITLE
M2O relation values in API differ from template & item detail apge

### DISCUSSION THREAD

**Discussion by eliteproxy7** [directus/directus#6586] [Archive]
I have a M2O relation between collectins Products < Brand

here is my set up
![image](https://user-images.githubusercontent.com/5486268/124017769-1a63fb00-d9b5-11eb-90ca-b76c219c90a5.png)

in my API endpoint for the item I see the fields showing the Id for the vendors > brand 

```
..
"brand_name": 6,
..
```

How can I have this show the text field brand_name i have specified in the brands collection item?

fyi: in the items detail, where the brand is associate i see the name. because I add it to the template.. it's just the API that is not showing the name only the associated ID

**Comment by rijkvanzanten** [comment]
![CleanShot 2021-06-30 at 15 53 51@2x](https://user-images.githubusercontent.com/9141017/124022879-63b74900-d9bb-11eb-883f-74644c922d38.png)

https://docs.directus.io/reference/api/items/#get-items

https://docs.directus.io/reference/api/query/#fields

https://docs.directus.io/reference/api/introduction/#relational-data
